test/pipe_test.go

```diff
@@ -105,6 +105,25 @@ func TestAcc_Pipe_InputFromAwsls(t *testing.T) {
 				"TOTAL NUMBER OF DELETED RESOURCES:",
 			},
 		},
+		{
+			name: "single resource with force",
+			awslsArgs: []string{
+				"-p", testVars.AWSProfile1,
+				"-r", testVars.AWSRegion1,
+				"-a", "tags", "aws_vpc"},
+			grepArgs:  []string{"foo"},
+			awsrmArgs: []string{"--force"},
+			expectedLogs: []string{
+				"PROCEEDING WITH DELETION AND SKIPPING CONFIRMATION \\(FORCE\\)",
```

Comment:
I guess the `\\` can be removed

Reply:
I added those since I saw them in other cases. But I removed them now. If you'd like me to remove the others I can as well.